container/go/cmd/pusher/pusher.go

```diff
@@ -32,18 +32,17 @@ import (
 
 var (
 	dst             = flag.String("dst", "", "The destination location including repo and digest/tag of the docker image to push. Supports fully-qualified tag or digest references.")
-	src             = flag.String("src", "", "Path to the config.json when -format is legacy, path to the index.json when -format is oci or path to the image .tar file when -format is docker.")
+	src             = flag.String("src", "", "Path to the index.json when -format is oci or path to the image .tar file when -format is docker, optional for legacy format.")
 	format          = flag.String("format", "", "The format of the source image, (oci, legacy, or docker). The docker format should be a tarball of the image as generated by docker save.")
 	clientConfigDir = flag.String("client-config-dir", "", "The path to the directory where the client configuration files are located. Overiddes the value from DOCKER_CONFIG.")
 	legacyBaseImage = flag.String("legacyBaseImage", "", "Optional: the path to a legacy base image in tarball form.")
+	configPath      = flag.String("configPath", "", "Optional: the path to the image config for legacy format.")
```

Comment:
This is not really `Optional`. I would say `Path to the image config. Should be specified only when format is legacy.`